runtime/logging.go

```diff
@@ -60,6 +60,7 @@ func (h *LoggingHandler) loggingEnabled(level logging.Level) bool {
 func (h *LoggingHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	var rctx logging.RequestContext
 	rctx.ReqID = atomic.AddUint64(&h.requestID, uint64(1))
+	rctx.HTTPRequestContext = logging.HTTPRequestContext{Header: r.Header.Clone()}
```

Comment:
Similar to the `requested_by` field in the decision log, the request context would be available in the log when the log-level is `info` or higher.